app/src/main/java/com/example/dine_and_donate/SignUpActivity.java

```diff
@@ -166,7 +166,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             mCreatedUser.isOrg = mSpinner.getSelectedItem().toString() == "Organization";
                             Toast.makeText(SignUpActivity.this, "Account created", Toast.LENGTH_SHORT).show();
                             writeNewUser(mUser.getUid(), mName.getText().toString(), email, mSpinner.getSelectedItem().toString().equals("Organization"));
-                            navigationHelper(ProfileActivity.class);
+                            //navigationHelper(ProfileActivity.class);
```

Comment:
?

Reply:
Commenting out ProfileActivity.class was just temporary so that the app wouldn't crashing after deleting ProfileActivity and replacing it with ProfileFragment. I was going to go back to fix these transitions to going from activity to fragment instead of using intents, but I will do that now and then create a new pull request.

Reply:
Oops nevermind! I didn't realize I could just do HomeActivity.class instead.